sdk/communication/Azure.Communication.Chat/README.md

```diff
@@ -91,12 +91,14 @@ Once you initialized a `ChatThreadClient` class, you can do the following chat o
 ### Update a thread
 
 ```C# Snippet:Azure_Communication_Chat_Tests_E2E_UpdateThread
-chatThreadClient.UpdateThread("Updated topic - C# sdk");
+string updatedTopic = "Updated topic - C# sdk";
+chatThreadClient.UpdateThread(updatedTopic);
```

Comment:
We can try to make it simpler by using named params. Then reads more English kinda like: "update thread set topic to Launch meeting".

``` C#
chatThreadClient.UpdateThread(topic: "Launch meeting");
```
Also trying to align it to a story would be easier to comprehend. We can make it about a hypothetical meeting:

 ``` C#
chatThreadClient.UpdateThread(topic: "Launch meeting");
chatThreadClient.SendMessage("Let's meet at 11am");
chatThreadClient.UpdateMessage(messageId, newContent: "Instead of 11am, let's meet at 2pm");
```
Note 1: I removed some of the optional parameters so make it simpler as the reader would have to think about fewer things.

Note 2: Changing the parameter name to `newContent` would be a breaking change. So let's keep it as `content` till we decide how we want to introduce these breaking changes.

Reply:
I like that idea of story telling, let me go over it to update it

Reply:
@DominikMe I removed the defaults on this one as suggested.